lib/css/components/_stacks-prose.less

```diff
@@ -14,7 +14,7 @@
     // Base styling
     font-size: @fs-body2;
     line-height: @lh-lg;
-    word-wrap: break-word;
+    word-wrap: break-all;
```

Comment:
This could end up having some weird ramifications by breaking words we don't want to be broken, but you're way closer to the problem than I am. Poke around at it, and if you feel good, feel free to merge.

Reply:
Yep, agreed. I'm playing around in the branch preview now

Reply:
Already seeing stuff I don't like